app/views/admin/npq/applications/_application_details.html.erb

```diff
@@ -0,0 +1,125 @@
+<%=
+  govuk_summary_list do |sl|
+    sl.row do |row|
+      row.key(text: 'Application UUID')
+      row.value(text: application.id || govuk_tag(text: "Missing", colour: "red"))
+      row.action
```

Comment:
Do you need an empty `.action` here? I thought [the component would handle these automatically](https://govuk-components.netlify.app/components/summary-list/#summary-cards).